src/gravel/controllers/orch/cephfs.py

```diff
@@ -82,3 +90,41 @@ def get_fs_info(self, name: str) -> CephFSListEntryModel:
             if fs.name == name:
                 return fs
         raise CephFSError(f"unknown filesystem {name}")
+
+    def authorize(
+        self,
+        fsname: str,
+        clientid: str
+    ) -> CephFSAuthorizationModel:
+        assert fsname and clientid
+        cmd = {
+            "prefix": "fs authorize",
+            "filesystem": fsname,
+            "entity": f"client.{fsname}-{clientid}",
+            "caps": ["/", "rw"],
+            "format": "json"
+        }
+        try:
+            res = self.mon.call(cmd)
+        except CephCommandError as e:
+            raise CephFSError(str(e)) from e
+        lst = parse_obj_as(List[CephFSAuthorizationModel], res)
+        assert len(lst) == 1
+        return lst[0]
+
+    def get_authorization(self, fsname: str) -> CephFSAuthorizationModel:
+        cmd = {
+            "prefix": "auth get",
+            "entity": f"client.{fsname}-default",
```

Comment:
This hardcodes `-default`, but `authorize()` takes clientid as a parameter, so there's a slight mismatch here. Should `get_authorization()` also have clientid as a parameter?

Reply:
yeah, good call. Let me rework this a little bit.